### PR TITLE
tools: tune: Generate ctrl bytes from params for DRC

### PR DESCRIPTION
**Pull request by johnylin76**
Used the similar parameter set as the DRC kernel in CRAS to generate ctrl bytes for DRC component.

Plotted input-output dB response curve with the specific parameter set.

### CODE REVIEW

**Comment by johnylin76** [comment]
Hi @cujomalainey @lgirdwood @singalsu , this PR is for DRC to add tune tools for generating ctrl bytes from the similar parameter set as the DRC implementation in CRAS. Please help to review, thanks.

File: tools/tune/drc/drc_gen_coefs.m [hunk 1]
```diff
@@ -0,0 +1,117 @@
+function coefs = drc_gen_coefs(params, sample_rate);
+
+pkg load control;
```
Comment (singalsu) [inline tools/tune/drc/drc_gen_coefs.m:3]: The Octave specifics those would result to error when running in Matlab are best to be placed inside 

```
if exist('OCTAVE_VERSION', 'builtin')
end
```

File: tools/tune/drc/example_drc.m [hunk 2]
```diff
@@ -1,18 +1,42 @@
...
-blob_fn = "../../ctl/drc_coef.blob" % Blob binary file
-alsa_fn = "../../ctl/drc_coef.txt" % ALSA CSV format file
+blob_fn = "../../ctl/drc_coef.blob"; % Blob binary file
+alsa_fn = "../../ctl/drc_coef.txt"; % ALSA CSV format file
 
 endian = "little";
 
-% DRC enabled
-enabled = 1
+sample_rate = 44100;
```
Comment (singalsu) [inline tools/tune/drc/example_drc.m:11]: Probably 48 kHz is better assumption for SOF?

**Review by singalsu — CHANGES_REQUESTED** [review]
Looks good, the minor add for Matlab compatibility would be good (control toolbox is there without initialization code if have licence for it). The DRC also probably is in the fixed 48 kHz rate side of a pipeline so it should be better assumption than 44.1 kHz config.

File: tools/tune/drc/example_drc.m (hunk 2 above)
Comment (cujomalainey) [reply tools/tune/drc/example_drc.m:11]: Yes and we use 48000 in CRAS, this should be changed

**Review by singalsu — APPROVED** [review]
Looks good, thanks Johnny!

**Review by cujomalainey — APPROVED** [review]

**Pull request merged**